docs/topics/multiplatform/multiplatform-native-artifacts.md

```diff
@@ -0,0 +1,370 @@
+[//]: # (title: Build Kotlin/Native artifacts)
+
+> The new DSL described below is [Experimental](components-stability.md). It may be changed at any time.
+> We encourage you to use it for evaluation purposes.
+> 
+{type="warning"}
+
+Kotlin/Native [targets](multiplatform-dsl-reference.md#native-targets) are compiled to the `*.klib` library artifacts,
+which can be consumed by Kotlin/Native itself as a dependency but cannot used as a native library.
+ 
+To declare final native binaries, use the new binaries format with the `kotlinArtifacts` DSL. It represents a collection
+of native binaries built for this target in addition to the default `*.klib` artifact and provides a set of methods for declaring and configuring them.
+ 
+> The `kotlin-multiplatform` plugin doesn't create any production binaries by default. The only binary available by default 
+> is a debug test executable that lets you run unit tests from the `test` compilation.
+>
+{type="note"}
+
+## Declare binaries
+
+Use the following kinds of binaries to declare elements of the `kotlinArtifacts` DSL: 
+
+| Factory method | Binary kind             | Available for                                |
+|----------------|-------------------------|----------------------------------------------|
+| `sharedLib`    | Shared native library   | All native targets, except for `WebAssembly` |
+| `staticLib`    | Static native library   | All native targets, except for `WebAssembly` |
+| `framework`    | Objective-C framework   | macOS, iOS, watchOS, and tvOS targets only   |
+| `fatFramework` | Universal fat framework | macOS, iOS, watchOS, and tvOS targets only   |
+| `XCFramework`  | XCFramework framework   | macOS, iOS, watchOS, and tvOS targets only   |
+
+The simplest version requires the `target` (or `targets`) and `modes` parameters for the selected build type. Currently, 
```

Comment:
Only target (or targets) is required.
Modes have default value equal `modes(DEBUG, RELEASE)`